### PR TITLE
Test/54/question editor component

### LINKED ISSUE #54: (SP: 1) Write tests for question editor component
Writing tests don't forget to mock elements if neccessary.

- [ ] it should renders question input field
- [ ] it should renders a open answer
- [ ] it should change question type
- [ ] it should change question and answer input fields
- [ ] it should click on edit title and category

### PR DESCRIPTION
**Pull request by poggy33**
(SP: 1)[#54] Tests for Question Editor Component

### CODE REVIEW

**Comment by nadraharr** [comment]
You need to resolve the conflicts:
~ git checkout develop
~ git pull
~ git checkout "your branch"
~ git merge develop
Then you can deside which version to save in code editor. Then push again.

**Review by nadraharr — APPROVED** [review]
Please, link your issues to this pull request.

**Review by lublubczyk — APPROVED** [review]

**Review by natamrshn — CHANGES_REQUESTED** [review]
please, resolve conflicts on branch

**Comment by poggy33** [comment]
> please, resolve conflicts on branch

I've resolved conflicts. Please, check.

**Review by zim89 — APPROVED** [review]

File: src/tests/unit/components/question-editor/QuestionEditor.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      name: 'questionPage.question'
+    })
+
+    const answerInput = screen.getByRole('textbox', {
+      name: 'questionPage.answer'
+    })
+
+    fireEvent.change(questionInput, { target: { value: 'Updated Question' } })
+
+    fireEvent.change(answerInput, { target: { value: 'Updated Answer' } })
```
Comment (vlad-khrychov) [inline src/tests/unit/components/question-editor/QuestionEditor.spec.jsx:82]: Try using `userEvent` instead.

File: src/tests/unit/components/question-editor/QuestionEditor.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+describe('QuestionEditor Component', () => {
+  beforeEach(() => {
+    render(
+      <QuestionEditor
+        data={mockData}
+        handleInputChange={handleInputChangeMock}
+        handleNonInputValueChange={handleNonInputValueChangeMock}
+        isQuizQuestion
+        loading={false}
```
Comment (vlad-khrychov) [inline src/tests/unit/components/question-editor/QuestionEditor.spec.jsx:23]: Maybe it's a good idea to add tests for these props as well.

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**